<?xml version="1.0"?>
<launch>

  <!--Planning parameter-->
  <arg name="planning_plugin" value="ompl_interface/OMPLPlanner" />
  <arg name="planning_adapters" value="default_planner_request_adapters/AddTimeParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints" />
  
  <!--Node that read data from LeapMotion and publish it -->
  <node name="leap_motion" pkg="leap_motion" type="sender.py" respawn="false" />
  
  <!--Main Node that take make the Moveit Planning-->
  <include file="$(find schunk_gripper_moveit_config)/launch/demo.launch"/>
 <!-- <node name="listener" pkg="sensor_listener" type="listener_learning" respawn="false" output="screen" args="600 kinematic">-->
 <node name="listener" pkg="sensor_listener" type="listener_learning" respawn="false" output="screen" args="kinematic_functions.py ikine" />
    <param name="planning_plugin" value="$(arg planning_plugin)" />
    <param name="request_adapters" value="$(arg planning_adapters)" />
    <param name="start_state_max_bounds_error" value="0.1" />
  </node>
  
  <!-- Start the Security pedal Node-->
	<node pkg="sensor_listener" type="pedal_security.py" name="security_pedal_node" respawn="true" output="screen"/>
	
  


<!--

<include file="$(find pr2_moveit_config)/launch/demo.launch"/>
<node name="move_group_interface_tutorial" pkg="pr2_moveit_tutorials" type="move_group_interface_tutorial" respawn="false" output="screen">
</node>

-->

</launch>
